politeiawww/comments.go

```diff
@@ -5,10 +5,95 @@
 package main
 
 import (
+	"github.com/decred/politeia/plugins/comments"
+	pi "github.com/decred/politeia/politeiawww/api/pi/v1"
 	www "github.com/decred/politeia/politeiawww/api/www/v1"
 	"github.com/decred/politeia/politeiawww/user"
+	"github.com/google/uuid"
 )
 
+func convertCommentsPluginPropStateFromPi(s pi.PropStateT) comments.StateT {
+	switch s {
+	case pi.PropStateUnvetted:
+		return comments.StateUnvetted
+	case pi.PropStateVetted:
+		return comments.StateVetted
+	}
+	return comments.StateInvalid
+}
+
+func (p *politeiawww) processComments(c pi.Comments) (*pi.CommentsReply, error) {
```

Comment:
Put all the pi api calls in `piwww.go` for now.  We'll reorganize it later once all the old code is deleted.  Just the comment plugin calls go in `comments.go`.